src/Sulu/Bundle/MediaBundle/Controller/MediaRedirectController.php

```diff
@@ -48,8 +49,12 @@ public function redirectAction(Request $request, $id)
         $locale = $this->getRequestParameter($request, 'locale', true);
         $format = $this->getRequestParameter($request, 'format');
 
-        /** @var Media $media */
-        $media = $this->mediaManager->getById($id, $locale);
+        try {
+            /** @var Media $media */
+            $media = $this->mediaManager->getById($id, $locale);
+        } catch (MediaNotFoundException $e) {
+            throw new NotFoundHttpException($e->getMessage());
```

Comment:
@Jupi007 the $previous exception should be given to the new exception.

```suggestion
            throw new NotFoundHttpException($e->getMessage(), $e);
```

Reply:
Oh, indeed. Sorry.
Thanks for the fix.